src/features/unified-share-modal/__tests__/__snapshots__/SharedLinkSection.test.js.snap

```diff
@@ -28,30 +25,18 @@ exports[`features/unified-share-modal/SharedLinkSection should account for share
         isOn={true}
         label={
           <span>
-            <FormattedMessage
-              defaultMessage="Shared link is created"
-              id="boxui.unifiedShare.linkShareOn"
-            />
+            Shared link is created
             <Tooltip
               constrainToScrollParent={false}
               constrainToWindow={true}
               isDisabled={false}
               position="top-center"
-              text={
-                <FormattedMessage
-                  defaultMessage="This link will expire and be inaccessible on {expiration, date, long}."
-                  id="boxui.unifiedShare.sharedLinkExpirationTooltip"
-                  values={
-                    Object {
-                      "expiration": 1519404618000000,
-                    }
-                  }
-                />
-              }
+              text="This link will expire and be inaccessible on {expiration, date, long}."
```

Comment:
Should we change this default message to be more generic? (I don't think `{expiration, date, long}` will be replaced by a human-readable date, will it??)

Reply:
that is a good point - I think this is worth addressing, but probably in a separate ticket